scripts/sync_check/Dockerfile-tester

```diff
@@ -1,6 +1,15 @@
 FROM fedora:36
 
-RUN dnf install -y docker ruby ruby-devel make gcc
+# Install dependencies
+RUN dnf install -y \
+    docker \
+    ruby \
+    ruby-devel \
+    make \
+    gcc \
+    && dnf clean all
 
 # Install required Ruby packages
-RUN gem install docker-api slack-ruby-client
+RUN gem install \
+    docker-api \
+    slack-ruby-client
```

Comment:
Same. Alternatively we do it in a separate issue https://github.com/ChainSafe/forest-iac/issues/34